starlark-test/tests/go-testcases/assign.sky

```diff
@@ -148,7 +148,7 @@ f()
 
 ---
 
-z += 3 ### Variable was not found
+z += 3         ### Augmented assignment is a binding and not allowed on a global variable
```

Comment:
nit: why the indentation before the comment?

Reply:
Because of the same indentation in the previous test in this file.

I'll revert indentation.